tutorials/xr/openxr_render_models.rst

```diff
@@ -0,0 +1,274 @@
+being pressed.
+
+For those runtimes that support the
+`controller data source for hand tracking <https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_EXT_hand_tracking_data_source>`_
+, we can also correctly position the user's fingers and hand according to the
+shape of the controller.
+Do note that this works in combination with the
+`hand joints motion range extension <https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_EXT_hand_joints_motion_range>`_
+to prevent clipping of the fingers.
+
+OpenXR Render models node
+-------------------------
+
+The :ref:`OpenXRRenderModelManager<class_OpenXRRenderModelManager>`
+node can be used to automate most of the render models functionality. 
+This node keeps track of the active render models currently made
+available by the XR runtime.
+
+It will create child nodes for each active render model resulting in
+that render model being displayed.
+
+This node must have an :ref:`XROrigin3D<class_XROrigin3D>` node as an
+ancestor.
+
+If ``tracker`` is set to ``Any`` our node will show all render models
+currently being tracked. In this scenario this node must be a direct
+child of our :ref:`XROrigin3D<class_XROrigin3D>` node.
+
+If ``tracker`` is set to ``None set`` our node will only show render
+models for which no tracker has been identified. In this scenario this
+node must also be a direct child of our
+:ref:`XROrigin3D<class_XROrigin3D>` node.
+
+If ``tracker`` is set to ``Left Hand`` or ``Right Hand`` our node will
+only show render models related to our left or right hand respectively.
+In this scenario, our node can be placed deeper in the scene tree.
+
+.. warning::
+
+    For most XR runtimes this means the render model represents a controller
+    that is actually being held by the user but this is not a guarantee.
+    Some XR runtimes will always set the tracker to either the left or right
+    hand even if the controller is not currently held but is being tracked.
+    You should always test this as this will lead to unwanted behavior.
+
+In this scenario we can also specify an action for a pose in the action map
+by setting the ``make_local_to_pose`` property to the pose action.
+Use this in combination with an :ref:`XRController3D<class_XRController3D>`
+node that is using the same pose and you can now add a layer that allows
+you to deviate from the tracked position of both your controller and the
+related render model (see example below).
+
+.. note::
+
+    Combining the above with hand tracking does introduce the problem
+    that hand tracking is completely independent from the action map
+    system. You will need to combine the hand tracking and controller
+    tracking poses to properly offset the render models.
+
+    This falls beyond the scope of this documentation but an example
+    implementation of this can be found in
+    `our hand tracking demo <https://github.com/godotengine/godot-demo-projects/tree/master/xr/openxr_render_models>`_.
```

Comment:
@BastiaanOlij does the hand tracking demo as it is now do what's described in this note? I'm not seeing any updates for that demo project for render models.

Reply:
Oh damn, I updated the URL but not the description, let me fix that.

Reply:
Ok, changed this a little because the original didn't make sense anymore. So the link has moved to the next section as the demo is what was used to create the rest of the documentation.